index.html

```diff
@@ -1144,11 +1107,18 @@ <h2>Dereferencing the Secondary Resource</h2>
 					case when dereferencing it returns an HTTP <code>3xx</code> (Redirection) response with a
 					<code>Location</code> header (see section 7.1.2 of [[RFC7231]].
 					</p>
-					</li>
 				</ol>
-				<li>Otherwise, dereference the secondary resource as defined by the media type ([[RFC2046]]) of the primary resource.
-				</li>
-				</li>
+				<li>Otherwise, dereference the secondary resource identified by the DID fragment as defined by the media type ([[RFC2046]]) of the primary resource.
+				For example, if the primary resource is a representation of a DID document with media type <code>application/did+ld+json</code>, then
```

Comment:
Note that multiple `+` have been disallowed in IETF/IANA media types. 

`application/ld+json` might be sufficient, here?
```suggestion
				For example, if the primary resource is a representation of a DID document with media type <code>application/ld+json</code>, then
```

Reply:
@TallTed I agreed with your comment, but I think it would be better if this gets discussed and fixed first in did-core, and then later in did-resolution, and that we address this in a separate PR.

Reply:
See https://github.com/w3c/did-core/issues/863